requirements-dev.txt

```diff
@@ -1,7 +1,6 @@
 # include requirements.txt so pip has context to avoid installing incompatible dependencies
 -r requirements.txt
 pytest
-ftfy
```

Comment:
without ftfy, slow HF tokenizer can still the correct result in CI pipeline?

Reply:
yes, but locally it will throw issues if it's not installed - hence the warning so when we're developing we don't have issues.